blueman/main/Manager.py

```diff
@@ -168,8 +173,12 @@ def prop_changed(lst, adapter, key_value):
             key, value = key_value
             if key == "Discovering" and not value:
                 prog.finalize()
-                self.List.disconnect_by_func(on_progress)
-                self.List.disconnect_by_func(prop_changed)
+                # FIXME for some reason the signal handler is None
```

Comment:
Just stumbled across this while adding loads of type hints. The reason is that `ManagerProgressbar` overrides `connect` with a version that does not return anything. We could make it return the obtained handle, however it stores it internally to disconnect it in the `finalize` method, so I think those two `disconnect` calls would compete and I'm not sure how that'd work out. :thinking: